README.md

```diff
@@ -172,17 +172,21 @@ err := formatter.Format(w, style, iterator)
 <a id="markdown-the-html-formatter" name="the-html-formatter"></a>
 ### The HTML formatter
 
-By default the `html` registered formatter generates standalone HTML with
```

Comment:
Please revert unnecessary grammatical changes.

Reply:
You mean README.md should have just one extra line about the new option?

Reply:
Yes please.